.github/workflows/ci.yml

```diff
@@ -23,7 +23,8 @@ jobs:
         cd -
     - name: V doctor
       run: v doctor
-
+    - name: format
```

Comment:
The CI isn't supposed to try to format for you (since that won't affect what goes in the repo, anyway).  The next section checks to make sure that what you pushed was already formatted, and that's what we want.  If it doesn't pass the next block, you should format in your playpen then push the formatted code.

Reply:
how do you do that?